builtin/game/chatcommands.lua

```diff
@@ -815,12 +815,18 @@ core.register_chatcommand("days", {
 })
 
 core.register_chatcommand("shutdown", {
+    params = "<delay_in_seconds(0..inf)> | Leave blank for normal shutdown",
```

Comment:
the only problem i see with the previous code is that we can do 3h2m3h and this will do 6h2m. Maybe a more stricter regexp (%dh)?(%dm)? ... is better